packages/react-three-rapier/src/Physics.tsx

```diff
@@ -274,54 +274,79 @@ export const Physics: FC<RapierWorldProps> = ({
       const collider1 = world.getCollider(handle1);
       const collider2 = world.getCollider(handle2);
 
-      const rigidBodyHandle1 = collider1.parent()?.handle;
-      const rigidBodyHandle2 = collider2.parent()?.handle;
-
-      if (!collider1 || !collider2 || rigidBodyHandle1 === undefined || rigidBodyHandle2 === undefined) {
+      if (!collider1 || !collider2) {
```

Comment:
I found that I couldn't trigger collision events on colliders which didn't have rigid bodies. The only reason I could see for this was that the event below requires a rigid body in the payload.

If we remove that I think we can enable collision events on standalone colliders.